### PR TITLE
Doc swagger for new history route

### PR DESCRIPTION
**Pull request by kun2985**
# Description

This PR doc swagger for new history route and a little correction for localization

## Issue (optional)

Issue link: bot-1315

## How to test
 - http://petstore.swagger.io/?url=https://raw.githubusercontent.com/CanalTP/Chaos/task-bot-1315-doc-swagger-route-history/documentation/swagger.yml#/WIP/get_disruptions__id__history

### CODE REVIEW

File: documentation/swagger.yml [hunk 1]
```diff
@@ -63,6 +63,7 @@
     - name: Traffic Reports
     - name: Search
     - name: Export
+    - name: WIP
```
Comment (pthegner) [inline documentation/swagger.yml]:
```suggestion
    - name: WIP (Work In Progress)
```

File: documentation/swagger.yml [hunk 2]
```diff
@@ -1252,6 +1253,46 @@
             description: Service unavailable
             schema:
               $ref: "#/definitions/error_occured"
+
+    /disruptions/{id}/history:
+      get:
+        description: Return all histories of one disruption
```
Comment (pthegner) [inline documentation/swagger.yml]:
```suggestion
        description: Return all versions of one disruption
```

File: documentation/swagger.yml [hunk 3]
```diff
@@ -2299,7 +2340,7 @@
             properties:
               id:
                 type: string
-                format: uuid
+                format: string
```
Comment (pthegner) [inline documentation/swagger.yml]: not necessary

File: documentation/swagger.yml [hunk 4]
```diff
@@ -2405,7 +2446,7 @@
...
```
Comment (pthegner) [inline documentation/swagger.yml]: not necessary

File: documentation/swagger.yml [hunk 5]
```diff
@@ -1252,6 +1253,46 @@
...
+        - $ref: "#/parameters/header_coverage"
+        - $ref: "#/parameters/header_customer_id"
+        - $ref: "#/parameters/header_contributors"
+        - name: id
+          in: path
+          required: true
+          type: string
+          description: Entity identifier
+        tags:
+          - WIP
```
Comment (pthegner) [inline documentation/swagger.yml]:
```suggestion
          - WIP (Work In Progress)
```

File: documentation/swagger.yml (hunk 3 above)
Comment (pthegner) [inline documentation/swagger.yml]:
```suggestion
```

File: documentation/swagger.yml (hunk 4 above)
Comment (pthegner) [inline documentation/swagger.yml]:
```suggestion
```

File: documentation/swagger.yml (hunk 2 above)
Comment (dvdn) [inline documentation/swagger.yml]:
```suggestion
        description: Get a list of historical states of a disruption
```

File: documentation/swagger.yml [hunk 6]
```diff
@@ -63,6 +63,7 @@
     - name: Traffic Reports
     - name: Search
     - name: Export
+    - name: WIP (Work In Progress)
```
Comment (dvdn) [inline documentation/swagger.yml]:
```suggestion
    - name: History (Work In Progress)
```

File: documentation/swagger.yml [hunk 7]
```diff
@@ -1252,6 +1253,46 @@
...
+        - $ref: "#/parameters/header_coverage"
+        - $ref: "#/parameters/header_customer_id"
+        - $ref: "#/parameters/header_contributors"
+        - name: id
+          in: path
+          required: true
+          type: string
+          description: Entity identifier
+        tags:
+          - WIP (Work In Progress)
```
Comment (dvdn) [inline documentation/swagger.yml]:
```suggestion
          - History (Work In Progress)
```

**Pull request merged**